packages/react-components/react-storybook-addon/.storybook/main.js

```diff
@@ -3,7 +3,7 @@ const rootMain = require('../../../../.storybook/main');
 module.exports = /** @type {Omit<import('../../../../.storybook/main'), 'typescript'|'babel'>} */ ({
   ...rootMain,
   stories: [...rootMain.stories, '../src/**/*.stories.mdx', '../src/**/*.stories.@(ts|tsx)'],
-  addons: ['../preset.js', ...rootMain.addons],
```

Comment:
removed relative path to unify how things work. in this case if something would stop working in root storybook config, we would get the same "issue" in our custom addon